benchmark/Resp.benchmark/RespOnlineBench.cs

```diff
@@ -323,7 +323,9 @@ public void Run()
                             $"{"total_ops;",pad}" +
                             $"{"iter_tops;",pad}" +
                             $"{"tpt (Kops/sec)",pad}";
+#pragma warning disable CA2254 // Template should be a static expression
```

Comment:
instead of disabling, could we rewrite this (and the other disable sites) as `logger?.LogInformation("{msg}", msg)`

Reply:
nice!